gobblin-restli/gobblin-throttling-service/gobblin-throttling-service-api/src/main/snapshot/org.apache.gobblin.restli.throttling.permits.snapshot.json

```diff
@@ -17,6 +17,18 @@
       "type" : "long",
       "doc" : "Client should not try to acquire permits before this delay has passed.",
       "optional" : true
+    }, {
```

Comment:
Is this changes from other PR? Can you rebase the newest changes so that it won't be included?

Reply:
Every time I build the project, this change will be automatically made.

Reply:
Got the point. Just mention in the description that why this changes (which is unrelated to the PR itself) shows up as part of code change.